src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java

```diff
@@ -318,7 +319,7 @@ public boolean onClick(Player p, int slot, ItemStack item, ClickAction action) {
 			}
 		});
 		
-		menu.addItem(4, new CustomItem(new MaterialData(Material.NOTE_BLOCK), "�ePreview", "�rSelected Sound: �b" + cfg.getString(sound + ".enum"), "�rVolume: �b" + cfg.getFloat(sound + ".volume"), "�rPitch: �b" + cfg.getFloat(sound + ".pitch"), "", "�7\u21E8 Click to hear a Preview"));
+		menu.addItem(4, new CustomItem(new MaterialData(Material.NOTE_BLOCK), "�ePreview", "�rSelected Sound: �b" + cfg.getString(sound + ".enum"), "�rVolume: �b" + cfg.getFloat(sound + ".volume"), "�rPitch: �b" + cfg.getFloat(sound + ".pitch"), "", "�7\u21E8 Click to hear a Preview"));
```

Comment:
Malformed characters in this class.
Use the '&' character instead.

Reply:
I will fix this